docs/src/catalyst_functionality/dsl_description.md

```diff
@@ -262,7 +264,8 @@ rn = @reaction_network begin
 end
 ```
 ```@example tut2
-convert(ODESystem, rn)
```

Comment:
Same comment.

Reply:
Generally, I don't see why we in docs should use `latexify(rn; form=:ode)` given that it is internally doing a `convert`, and explicitly just doing convert both shows the equations and shows the user how to get the ODEs.

Reply:
I think we should use `latexify(rn; form=:ode)` rather than `latexify(convert(ODESystem,rn))` for the same reason we do
`ODEProblem(rn,...)` rather than `ODEProblem(convert(ODESystem,rn),...)`

(right now we might actually do some additional stuff in this example, but at some point `ODEProblem(rn,...)`and `ODEProblem(convert(ODESystem,rn),...)` would have been equivalent, and the first one was preferred, and in my mind the latexify case is very similar)